src/ansys/fluent/interface/post_objects/post_object_definitions.py

```diff
@@ -418,7 +418,22 @@ def allowed_values(self):
     class filled(metaclass=PyLocalPropertyMeta):
         """Draw filled contour."""
 
-        value: bool = True
+        _value: bool = True
```

Comment:
- Fluent behaviour:
    - filled = True:
        - node_values is active & can be either True or False
    - filled = False:
        - node_values is inactive with **implicit** value True
- We can produce the same behaviour here.  No exceptions are required.
- The accessor for node_values shall separate the explicit value from the implicit value. No other special handling is needed.